include/nbl/asset/IMesh.h

```diff
@@ -22,6 +23,7 @@ namespace asset
 		protected:
 			//! The cached bounding box of this mesh
 			core::aabbox3d<float> m_cachedBoundingBox;
+      core::OBB m_orientedBoundingBox;
```

Comment:
don't add an OBB to a mesh

Reply:
only seemed logical to follow the pattern, as in the example test, like the aabb which was already calculated on loading asset, could get the obb as well. Though, I designed obb to be disabled if not needed anyway.